project/cmake/installdata/common/common.txt

```diff
@@ -3,11 +3,14 @@ sounds/*
 system/keymaps/*
 system/library/*
 system/players/VideoPlayer/etc/*
+system/python/*
 system/shaders/*
 system/settings/*
 userdata/*
 system/addon-manifest.xml
 system/colors.xml
+system/IRSSmap.xml
 system/peripherals.xml
 system/playercorefactory.xml
+system/X10-Lola-IRSSmap.xml
```

Comment:
X10-Lola-IRSSmap.xml and IRSSmap.xml are used in `project/cmake/installdata/windows/irss.txt`. If we need them everywhere, could you drop them there?